tests/integration_tests/scheduler/mock_torque_bin/runner

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+job=$1
+
+function handle_sigterm {
+    # Torque uses (256 + SIGNAL) as the returncode
+    echo "271" > "${job}.returncode"
```

Comment:
write to a torque-specific filename, so that the lsf and slurm PR's can utilize this same runner and just let it write to more returncode files as needed.

Reply:
It's better for the people who will implement LSF or SLURM to modify this script rather than me guessing what they want in the future. Maybe add an environment variable which determines what sigterm should write to returncode, and this can be set by `qsub`, `bsub` etc.